index.html

```diff
@@ -10,7 +10,68 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+
+<section>
+    <header>
+    <span id="text_weather">
+    <h1 > Weather Report</h1>
+        <span id="for_city"> <h3>For the city of</h3>
+            <div id="output">Seattle</div>
+        </span>
+        
+    </span>
+    </header>
+
+    <section>
+        <h2>Garden</h2>
+    <div class="garden" id="garden_content">
+        <div id="sky-emotes">☁☼☁☼☁☼☁☼☁☼☁☼☁</div>
+        <div id="landscape">🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷</div>
+
+    </div>
+    </section>
+</section>
+
+
+    <section class="container">
+        <section id="temp">
+        <h2>Temperature</h2>
+        <div class="tempControls"></div>
+            <button id="increaseTempButton">🔼</button>
+            <span id="defaultTemp" class="orange">72</span>
+            <button id="decreaseTempButton">🔽</button>
+        <div>
+            <button id="realTimeTempButton">Real Time Temperature</button>
+        </div>
+        </section>
+
+        <section id="sky" onclick="resetSky()">
```

Comment:
It's a little confusing if some elements have an `onclick` defined and some elements have their click handlers registered for in the .js file (in `registerEventHandlers`). It's probably better to pick one or the other.